instrumentation/rack/lib/opentelemetry/instrumentation/rack/middlewares/tracer_middleware.rb

```diff
@@ -53,7 +53,7 @@ def initialize(app)
           end
 
           def call(env) # rubocop:disable Metrics/AbcSize, Metrics/MethodLength
-            if untraced_request?(env)
+            if untraced_request?(env) || filtered_request?(env)
```

Comment:
I think we can combine the two methods, so this remains:
```suggestion
            if untraced_request?(env)
```